README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# hw01
```

Comment:
It's best practice to include line breaks after headers. It makes the source file easier to read and will prevent rendering errors in the future.

```
# hw01

## About me

## My name is Jared and I'm from Colorado. 

### My Hobbies

* I play **guitar** and love Rock and Alternative music
* I love science fiction and fantasy stories
  + My favorite book is *Hitchhiker's Guide to the Galaxy*
* I am a big fan of movies and video games
  + Some of my favorite movies are *The Matrix* and *Monty Python and the Holy Grail*
* I act and have been doing mainly Shakespeare plays lately
```

# hw01

## About me

## My name is Jared and I'm from Colorado. 

### My Hobbies

* I play **guitar** and love Rock and Alternative music
* I love science fiction and fantasy stories
  + My favorite book is *Hitchhiker's Guide to the Galaxy*
* I am a big fan of movies and video games
  + Some of my favorite movies are *The Matrix* and *Monty Python and the Holy Grail*
* I act and have been doing mainly Shakespeare plays lately